python/ribasim_testmodels/ribasim_testmodels/trivial.py

```diff
@@ -97,8 +97,10 @@ def trivial_model() -> ribasim.Model:
     )
 
     model = ribasim.Model(
-        node=node,
-        edge=edge,
+        database=ribasim.Database(
+            node=node,
+            edge=edge,
+        ),
```

Comment:
I like that these are now together in a single class, and understand you named it `Database` since these tables always have to be in the database. But wouldn't a name like `Network` better represent what it really is? If all tables are stored in the database, users may expect to find all tables in `ribasim.Database`.

Reply:
The field is database because it is named so in the toml. I thought about naming the class Topology, but Network is a good alternative. This can be renamed in the follow up PR, when we actually start using said class more extensively.